PeerioTechnologies/peerio-mobile
# Feat update settings menu

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/10812/mobile-avatar-setting-menu
https://github.com/PeerioTechnologies/peerio-icebear/pull/251

#### Testing instructions  
Verify that new settings menu (just the first set of settings) match the mocks.
Verify that the payment / upgrade flow is still working as intended.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
